app/package.json

```diff
@@ -25,15 +25,19 @@
     "@babel/preset-env": "^7.16.11",
     "@vue/cli-plugin-eslint": "^5.0.8",
     "@vue/cli-plugin-router": "~5.0.8",
-    "@vue/cli-plugin-unit-jest": "~4.5.15",
-    "@vue/cli-service": "~4.5.15",
+    "@vue/cli-plugin-unit-jest": "~5.0.0",
+    "@vue/cli-service": "~5.0.8",
```

Comment:
I didn't understand how to resolve the dependencies of these package, but I solved it with vue add unit-jest. vue upgrade alone didn't solve it.